### PR TITLE
Add rest api

### PR DESCRIPTION
**Pull request by martmull**
add rest api routes :
- GET `/rest/<objectNamePlural>/<id>?filter=<FILTERS>&order_by=<ORDER_BY>&limit=<LIMIT>&depht=<DEPTH>&last_cursor=<LAST_CURSOR>`
- DELETE `/rest/<objectNamePlural>/<id>`

### CODE REVIEW

File: server/src/core/api-rest/api-rest.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+      ],
+    }).compile();
+
+    service = module.get<ApiRestService>(ApiRestService);
+  });
+  it('should be defined', () => {
+    expect(service).toBeDefined();
+  });
+  describe('checkFilterQuery', () => {
+    it('should check filter query', () => {
```
[the remark this answers is not in the document]
Comment (charlesBochet) [reply server/src/core/api-rest/api-rest.service.spec.ts:31]: We should still leave basic cases in this files to test the api-rest.service api, but keep all the edge cases for filter-parse.util.spec.ts

**Review by charlesBochet — COMMENTED** [review]
Left comments!

**Review by charlesBochet — APPROVED** [review]
Bravo!
